tests/test_Units.py

```diff
@@ -46,6 +46,8 @@ def test_units(xmlfile, csc, topic):
     for unit in root.findall(f"./{saltype}/item/Units"):
         if not unit.text:
             assert False, 'Units cannot be blank.'
+        elif " " in unit.text:
+            assert False, 'Units cannot have blank spaces'
```

Comment:
This check is not working properly, as it catches instances of `m / s` which is valid.  Instead I suggest modifying the first condition to be `if not unit.text.replace(" ", ""):` which will allow units formatted like `m / s` but will not allow fields of just spaces.

Reply:
"as it catches instances of m / s which is valid", geat point. Ok I have changed the test now.